chemprop/data/splitting.py

```diff
@@ -222,9 +233,15 @@ def split_component(
     match SplitType.get(split):
         case SplitType.CV_NO_VAL | SplitType.CV:
             # convert indices to datapoints for each fold
-            train = [[[datapoints[i] for i in fold] for datapoints in datapointss] for fold in train_idxs]
-            val = [[[datapoints[i] for i in fold] for datapoints in datapointss] for fold in val_idxs]
-            test = [[[datapoints[i] for i in fold] for datapoints in datapointss] for fold in test_idxs]
+            train = [
+                [[datapoints[i] for i in fold] for datapoints in datapointss] for fold in train_idxs
+            ]
+            val = [
+                [[datapoints[i] for i in fold] for datapoints in datapointss] for fold in val_idxs
+            ]
+            test = [
+                [[datapoints[i] for i in fold] for datapoints in datapointss] for fold in test_idxs
+            ]
         case SplitType.SCAFFOLD_BALANCED | SplitType.RANDOM_WITH_REPEATED_SMILES | SplitType.RANDOM | SplitType.KENNARD_STONE | SplitType.KMEANS:
```

Comment:
I'm surprised formatting with black didn't split this line into multiple lines. But that's okay to me.